test/onnx/verify_onnx.cpp

```diff
@@ -1245,6 +1245,42 @@ TEST_CASE(nonzero_test)
     EXPECT(migraphx::verify::verify_rms_range(result_vector, gold));
 }
 
+TEST_CASE(qlinearadd_test)
+{
+    // github.com/microsoft/onnxruntime/blob/main/docs/ContribOperators.md#com.microsoft.QLinearAdd
+    migraphx::program p = migraphx::parse_onnx("qlinearadd_test.onnx");
+    p.compile(migraphx::make_target("ref"));
+
+    migraphx::shape a{migraphx::shape::uint8_type, {64}};
+    std::vector<unsigned char> data_a = {
```

Comment:
`uint8_t`  should be available to use i think.

Reply:
Sure. (same thing underneath.., but will change it). Thanks.

Reply:
Done.

Reply:
I don't see it changed.

Reply:
Sorry, it was only partially changed.. Will fix it. Thanks.

Reply:
Done.